src/Uno.UI/UI/Xaml/Media/Animation/ColorAnimationUsingKeyFrames.Android.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace Windows.UI.Xaml.Media.Animation
+{
+	partial class ColorAnimationUsingKeyFrames
+	{
+		// For performance consideration, do not report each frame if we are GPU bound
+		// Frame will be reported on Pause or End (cf. InitializeAnimator)
+		private bool ReportEachFrame() => this.GetIsDependantAnimation() || this.GetIsDurationZero();
+
+		partial void OnFrame(IValueAnimator currentAnimator)
+		{
+			if (currentAnimator is CPUBoundAnimator<ColorOffset> cpuAnimator)
+			{
+				var easing = cpuAnimator.GetEasingFunction();
+				var newValue = easing.Ease(
+					currentAnimator.CurrentPlayTime,
+					_startingValue ?? default(ColorOffset),
+					_finalValue,
+					currentAnimator.Duration);
+
+				SetValue(newValue);
+				return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this redundant jump.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5444000)